Backpack-SwiftUI/BottomSheet/Classes/BPKBottomSheet.swift

```diff
@@ -41,6 +41,7 @@ public extension View {
         closeButtonAccessibilityLabel: String? = nil,
         title: String? = nil,
         action: BPKBottomSheetAction? = nil,
+        rootViewController: UIViewController?,
```

Comment:
left a comment below about the optionality of this, but also: Can we rename the property to be `presentingController` or similar to that? it would reflect what the VC will be used for. `rootViewController` alludes to what we used internally, but may not be the case for clients.

Reply:
Completely agree, much clearer!

Reply:
Updated 👍 
af27a9d4f4c7e6b9c82c4aa5bbbcbc299aec703c